<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Index_account</class>
 <widget class="QWidget" name="Index_account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Index_account</string>
  </property>
  <widget class="QFrame" name="frame_window">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>736</width>
     <height>366</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame[objectName=&quot;frame_window&quot;]
{
	border-radius:15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>695</x>
      <y>2</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color:white;
	border-top-right-radius:14px;
	font: 20pt &quot;Calibri&quot;;/*字体*/
	background-color: rgba(255, 255, 255,0);/*背景色*/
	border:0px;
}
QPushButton:hover
{
	background-color: rgba(255, 255, 255,0.5);
	color: rgb(255, 0, 0);
}

</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_minimize">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>2</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color:white;
	font: 20pt &quot;Calibri&quot;;/*字体*/
	background-color: rgba(255, 255, 255,0);/*背景色*/
	border:0px;
}
QPushButton:hover
{
	background-color: rgba(255, 255, 255,0.5);
	color: rgb(0, 0, 0)
}

</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>732</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;
color:rgb(20,20,20);
padding-bottom:1px;
border-top-left-radius:14px;
border-top-right-radius:14px;
background-color: rgb(110, 205, 255)</string>
    </property>
    <property name="text">
     <string>天堂证券中心</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListView" name="lv_members">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>75</y>
      <width>161</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView
{
	border:0px solid gray;
	border-radius:10px;
	background-color: rgb(250, 250, 250);
	outline:0px;
}

QListView::item
{
	height:25px;
	border-radius:10px;
}

QListView::item:hover
{
	background-color: rgb(157, 213, 255);
	height:25px;
	border-radius:10px;
}

QListView::item:selected
{
	background-color: rgb(85, 170, 255);
	border-radius:10px;
	color:black;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_search">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>40</y>
      <width>125</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px solid gray;
border-radius:7px;
padding-left:3px;
padding-bottom:2px;
font:13px;
background-color: rgb(250, 250, 250);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_freeze_account">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>45</y>
      <width>86</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:0px solid gray;
	border-radius:10px;
	background-color: rgb(232, 232, 232);
	color:blue;
	font:13px;
}
QPushButton::hover
{
	border:2px solid blue;
}
</string>
    </property>
    <property name="text">
     <string>冻结账户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_unfreeze_account">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>45</y>
      <width>86</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:0px solid gray;
	border-radius:10px;
	background-color: rgb(232, 232, 232);
	font:13px;
}
QPushButton::hover
{
	border:2px solid rgb(0, 170, 255);
}
</string>
    </property>
    <property name="text">
     <string>解冻账户</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_state_value">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green;
font:15px;</string>
    </property>
    <property name="text">
     <string>正常</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_money_value">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>97</y>
      <width>156</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:30px;
color:rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_money">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;</string>
    </property>
    <property name="text">
     <string>存款：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>98</y>
      <width>31</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:25px;
</string>
    </property>
    <property name="text">
     <string>￥</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_state">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;</string>
    </property>
    <property name="text">
     <string>账户状态：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_account">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>45</y>
      <width>86</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:0px solid gray;
	border-radius:10px;
	background-color: rgb(232, 232, 232);
	font:13px;
}
QPushButton::hover
{
	border:2px solid rgb(0, 170, 255);
}
</string>
    </property>
    <property name="text">
     <string>新增账户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove_account">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>45</y>
      <width>86</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:0px solid gray;
	border-radius:10px;
	background-color: rgb(232, 232, 232);
	color:red;
	font:13px;
}
QPushButton::hover
{
	border:2px solid red;
}
</string>
    </property>
    <property name="text">
     <string>删除账户</string>
    </property>
   </widget>
   <widget class="QChartView" name="chart_view" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>155</y>
      <width>531</width>
      <height>196</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;</string>
    </property>
   </widget>
   <zorder>lbl_title</zorder>
   <zorder>btn_close</zorder>
   <zorder>btn_minimize</zorder>
   <zorder>lv_members</zorder>
   <zorder>le_search</zorder>
   <zorder>btn_freeze_account</zorder>
   <zorder>btn_unfreeze_account</zorder>
   <zorder>lbl_state_value</zorder>
   <zorder>lbl_money_value</zorder>
   <zorder>lbl_money</zorder>
   <zorder>label_5</zorder>
   <zorder>lbl_state</zorder>
   <zorder>btn_add_account</zorder>
   <zorder>btn_remove_account</zorder>
   <zorder>chart_view</zorder>
  </widget>
  <widget class="QLabel" name="lbl_search">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>48</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(Resources/find_25x25.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame_blur">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>736</width>
     <height>366</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255, 255, 255,0.9);
border:2px solid gray;
border-radius:15px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <zorder>frame_blur</zorder>
  <zorder>frame_window</zorder>
  <zorder>lbl_search</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header location="global">qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btn_add_account</sender>
   <signal>clicked()</signal>
   <receiver>Index_account</receiver>
   <slot>btn_add_account_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_freeze_account</sender>
   <signal>clicked()</signal>
   <receiver>Index_account</receiver>
   <slot>btn_freeze_account_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_unfreeze_account</sender>
   <signal>clicked()</signal>
   <receiver>Index_account</receiver>
   <slot>btn_unfreeze_account_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>-8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_remove_account</sender>
   <signal>clicked()</signal>
   <receiver>Index_account</receiver>
   <slot>btn_remove_account_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>665</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>-18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lv_members</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>Index_account</receiver>
   <slot>lv_members_click(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>-19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_minimize</sender>
   <signal>clicked()</signal>
   <receiver>Index_account</receiver>
   <slot>btn_minimize_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>-11</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>Index_account</receiver>
   <slot>btn_close_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>727</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>-23</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_add_account_click()</slot>
  <slot>btn_freeze_account_click()</slot>
  <slot>btn_unfreeze_account_click()</slot>
  <slot>btn_remove_account_click()</slot>
  <slot>lv_members_click(QModelIndex)</slot>
  <slot>btn_minimize_click()</slot>
  <slot>btn_close_click()</slot>
 </slots>
</ui>
